### ISSUE TITLE
Better preview for content

### ISSUE THREAD

**Issue by btzr-io** [lbryio/lbry-desktop#620]
## Update
See https://github.com/lbryio/lbry-app/issues/620#issuecomment-375822739
## The Issue
The preview component seems to give priority for videos and 
this affects the rest of the published content ( files ) :disappointed: 

### Steps to reproduce
1. Publish a content that isn't a video, audio or image.
2. Go to the claim page and see the preview.

### Expected behaviour
Support multiple preview layouts ( or a better and clean generic viewer ) for the specific content type, and display and empty state when the app can't preview the content 
(non-supported videos / audio, binary files...)
See -> https://github.com/lbryio/lbry-app/issues/551

### Actual behaviour
Already explain this -> Poor `ux`
See the gif below...
## Screenshots

![really-bad-ux](https://user-images.githubusercontent.com/14793624/30841834-7a5667d8-a23b-11e7-9b6e-2c587e093a23.gif)

### Anything else
Related issue: https://github.com/lbryio/lbry-app/issues/482 https://github.com/lbryio/lbry-app/issues/563

**Comment by btzr-io** [comment]
The current preview of files sucks and has a really poor ux, but it's `render-media` fault :stuck_out_tongue:

**Comment by kauffj** [comment]
Render media is actually only like 350 lines of JavaScript, FWIW

**Comment by btzr-io** [comment]
the problem is that returns an iframe, so we don't have much control on the styles :upside_down_face:

**Comment by LavRadis** [comment]
Is it possible to have multiple preview modules for different content types? I discussed it with @btzr-io and he agrees. Maybe the dev team can add a simple one for odf, txt or pdf to enable preview of those file types. I just googled and found: http://viewerjs.org/ There are probably many more open sourced viewers that could be embedded to improve the user experience. Personally, I would like to see a 3d model preview (stl, obj) and some sort of gps data preview.
